src/ports/nfts/utils.ts

```diff
@@ -92,6 +92,14 @@ export function getFetchQuery(
     )
   }
 
+  if (filters.ids && filters.ids.length > 0) {
+    if (filters.ids.length === 1) {
+      where.push(`id: "${filters.ids[0]}"`)
```

Comment:
I don't think this is needed, the case where the `length` is 1 will also work with the `id_in` from below

Reply:
👍